packages/expect/src/__tests__/__snapshots__/spy_matchers.test.js.snap

```diff
@@ -1059,3 +1234,638 @@ exports[`toHaveBeenNthCalledWith works with trailing undefined arguments 1`] = `
 Expected mock function first call to have been called with:
   Did not expect argument 2 but it was called with <red>undefined</>."
 `;
+
+exports[`toHaveLastReturnedWith works only on spies or jest.fn 1`] = `
+"<dim>expect(</><red>jest.fn()</><dim>)[.not].toHaveLastReturnedWith(</><dim>)</>
+
+<red>jest.fn()</> value must be a mock function or spy.
+Received:
+  function: <red>[Function fn]</>"
+`;
+
+exports[`toHaveLastReturnedWith works when not called 1`] = `
+"<dim>expect(</><red>jest.fn()</><dim>).toHaveLastReturnedWith(</><green>expected</><dim>)</>
+
+Expected mock function to have last returned:
+  <green>\\"foo\\"</>
+But it last returned:
+  <red>undefined</>"
+`;
+
+exports[`toHaveLastReturnedWith works with Immutable.js objects directly created 1`] = `
+"<dim>expect(</><red>jest.fn()</><dim>).not.toHaveLastReturnedWith(</><green>expected</><dim>)</>
+
+Expected mock function to not have last returned:
+  <green>Immutable.Map {\\"a\\": {\\"b\\": \\"c\\"}}</>"
+`;
```

Comment:
should this say "but it did" or something?

Reply:
Yeah, updated

![](http://dp.hanlon.io/1e2I46401i3L/Image%202018-04-29%20at%202.39.03%20PM.png)